packages/ui/src/components/va-backtop/VaBacktop.stories.ts

```diff
@@ -1,12 +1,79 @@
-import { defineComponent } from 'vue'
-import VaBacktop from './VaBacktop.demo.vue'
+import { VaBacktop } from './'
+import { VaRadio } from '../va-radio'
 
 export default {
   title: 'VaBacktop',
   component: VaBacktop,
 }
 
-export const Default = defineComponent({
+export const Default = () => ({
   components: { VaBacktop },
-  template: '<VaBacktop/>',
+  template: `
+    <p class="m-1">{{$vb.lorem(10000)}}</p>
+    <va-backtop/>
+  `,
 })
+
+export const Color = () => ({
+  components: { VaBacktop },
+  template: `
+    <p class="m-1">{{$vb.lorem(10000)}}</p>
+    <va-backtop color="warning"/>
+  `,
+})
+
+export const Offset = () => ({
+  components: { VaBacktop },
+  template: `
+    <p class="m-1">{{$vb.lorem(10000)}}</p>
+    <va-backtop 
+      horizontalOffset="4rem" 
+      verticalOffset="4rem"
+    />
+  `,
+})
+
+const OPTIONS_X = ['left', 'right']
```

Comment:
I suggest we move these to data block. So that it's clear OPTIONS are going to be used only for that story and not in some other stories.


```
data: () => {
  const OPTIONS_X = ['left', 'right']
  const OPTIONS_Y = ['top', 'bottom']
  return { optionsY: OPTIONS_Y, valueY: OPTIONS_Y[1], optionsX: OPTIONS_X, valueX: OPTIONS_X[1]}
},
```